### PR TITLE
Add JCL

### PR DESCRIPTION
**Pull request by jilloestreicher**
## Adding a language

- [x] The code displays "Hello World" ([tio.run](https://tio.run) may help for testing)
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme
- [x] I have no association with the language
- [x] There are no copyright issues with this code
- [x] The language has not been added prior to this pull request

### CODE REVIEW

**Review by Richienb — CHANGES_REQUESTED** [review]
where code

**Comment by jilloestreicher** [comment]
code is the jcl.text file in the j folder

**Review by Richienb — CHANGES_REQUESTED** [review]
Should the extension be `.text`?

**Review by Richienb — CHANGES_REQUESTED** [review]
Remember to update the languages count

**Comment by jilloestreicher** [comment]
oh sorry, should be .jcl

**Comment by jilloestreicher** [comment]
I updated the languages count as well

**Review by Richienb — CHANGES_REQUESTED** [review]
Should the file name also be in all caps like the link text in the readme?

**Comment by jilloestreicher** [comment]
okay I updated it to JCL.jcl

**Review by Richienb — CHANGES_REQUESTED** [review]
The file name still needs to be updated

File: j/JCL.jcl [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//* PRINT "HELLO WORLD" ON JOB OUTPUT
+//*
+//* NOTE THAT THE EXCLAMATION POINT IS INVALID EBCDIC FOR JCL
+//*   AND WILL CAUSE A JCL ERROR
+//*
+//STEP0001 EXEC PGM=IEBGENER
+//SYSIN    DD DUMMY
+//SYSPRINT DD SYSOUT=*
+//SYSUT1   DD *
+HELLO, WORLD
```
Comment (Richienb) [inline j/JCL.jcl:13]: Wouldn't this [print `HELLO, WORLD` instead of `HELLO WORLD`](https://www.youtube.com/watch?v=XbuKDC_n3N4&t=649s)?

**Review by Richienb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**